Client/mods/deathmatch/logic/CClientVectorGraphicDisplay.cpp

```diff
@@ -0,0 +1,167 @@
+/*****************************************************************************
+ *
+ *  PROJECT:     Multi Theft Auto v1.0
+ *               (Shared logic for modifications)
+ *  LICENSE:     See LICENSE in the top level directory
+ *  FILE:        mods/shared_logic/CClientVectorGraphicDisplay.cpp
+ *  PURPOSE:     Client vector graphic display base class
+ *
+ *****************************************************************************/
+
+#include "StdInc.h"
+
+using std::list;
+
+float CClientVectorGraphicDisplay::m_fGlobalScale = 1.0f;
+
+CClientVectorGraphicDisplay::CClientVectorGraphicDisplay(CClientDisplayManager* pDisplayManager, CClientVectorGraphic* pVectorGraphic, int ID) : CClientDisplay(pDisplayManager, ID)
+{
+    // Init
+    m_pVectorGraphic = pVectorGraphic;
+    m_fScale = 1;
+    m_ulFormat = 0;
+    m_bVisible = true;
+    m_bIsCleared = false;
+
+    UpdateTexture();
+}
+
+CClientVectorGraphicDisplay::~CClientVectorGraphicDisplay()
+{
+}
+
+void CClientVectorGraphicDisplay::SetPosition(const CVector& vecPosition)
+{
+    m_vecPosition = vecPosition;
+}
+
+void CClientVectorGraphicDisplay::Render()
+{
+    if (!m_pVectorGraphic || m_pVectorGraphic->IsDestroyed())
+        return;
+
+    if (!m_bVisible)
+    {
+        if (!IsCleared())
+        {
+            ClearTexture();
+        }
+            
+        return;
+    }
+    else
+    {
+        if (IsCleared())
+        {
+            m_bIsCleared = false;
+        }
+    }
+
+    if (HasUpdated())
+        UpdateTexture();
+}
+
+
+
+void CClientVectorGraphicDisplay::UpdateTexture()
+{
+    if (m_pVectorGraphic->IsDestroyed())
+        return;
+
+    SVGDocument* svgDocument = m_pVectorGraphic->GetSVGDocument();
+
+    if(!m_pVectorGraphic->GetSVGDocument())
+        return;
+
+    CVectorGraphicItem* pVectorGraphicItem = m_pVectorGraphic->GetRenderItem();
+
+    IDirect3DSurface9* surface = m_pVectorGraphic->GetRenderItem()->m_pD3DRenderTargetSurface;
+
+    if (!surface)
+        return;
+
+    IDirect3DDevice9* device = pVectorGraphicItem->m_pDevice;
+
+    uint width = pVectorGraphicItem->m_uiSizeX;
+    uint height = pVectorGraphicItem->m_uiSizeY;
+
+    Bitmap bitmap = svgDocument->renderToBitmap(width, height, 96.0);
```

Comment:
`96.0`?

Reply:
96 dpi

Reply:
Would be good to add a comment for that (or even a constexpr constant)